src/components/DownloadResultButton.tsx

```diff
@@ -1,106 +1,88 @@
+import Radio from '@mui/material/Radio';
+import Stack from '@mui/material/Stack';
 
-const options = ['Download selected query results', 'Download selected query results with URIs'];
-
-function DownloadResultButton({
-  disabled,
-  handleClick,
-  loading,
-}: {
+type Props = {
   disabled: boolean;
   handleClick: (index: number) => void;
   loading: boolean;
-}) {
-  const [open, setOpen] = useState(false);
-  const anchorRef = useRef<HTMLDivElement>(null);
-  const [selectedIndex, setSelectedIndex] = useState(0);
+};
 
-  const handleMenuItemClick = (index: number) => {
-    setSelectedIndex(index);
-    setOpen(false);
-  };
+export default function DownloadResultButton({ disabled, handleClick, loading }: Props) {
+  const [selectedIndex, setSelectedIndex] = useState<number>(0);
 
-  const handleToggle = () => {
-    setOpen((prevOpen) => !prevOpen);
+  const onDownload = () => {
+    handleClick(selectedIndex);
   };
 
-  const handleClose = (event: Event) => {
-    if (anchorRef.current && anchorRef.current.contains(event.target as HTMLElement)) {
-      return;
-    }
-
-    setOpen(false);
+  const onRadioChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const val = (event.target as HTMLInputElement).value;
+    if (val === 'labels') setSelectedIndex(0);
+    else setSelectedIndex(1);
   };
 
-  const button = (
-    <>
-      <ButtonGroup disabled={disabled || loading} variant="contained" ref={anchorRef}>
-        <Button onClick={() => handleClick(selectedIndex)} data-cy="download-results-button">
+  const control = (
+    <Box sx={{ display: 'inline-block' }}>
+      <Stack spacing={1} alignItems="flex-start" sx={{ width: 'fit-content' }}>
+        <Button
+          variant="contained"
+          onClick={onDownload}
+          disabled={disabled || loading}
+          data-cy="download-results-button"
+          aria-label="Download selected query results"
+          sx={{ width: '100%', textTransform: 'none' }}
+        >
           {loading ? (
             <>
-              Downloading selected query results
+              Downloading...
               <CircularProgress size={16} color="inherit" sx={{ ml: 1 }} />
             </>
           ) : (
-            options[selectedIndex]
+            'Download'
           )}
         </Button>
-        <Button data-cy="download-results-dropdown-button" size="small" onClick={handleToggle}>
-          <ArrowDropDownIcon />
-        </Button>
-      </ButtonGroup>
-      <Popper sx={{ zIndex: 1 }} open={open} anchorEl={anchorRef.current} transition disablePortal>
-        {({ TransitionProps, placement }) => (
-          <Grow
-            // eslint-disable-next-line react/jsx-props-no-spreading
-            {...TransitionProps}
-            style={{
-              transformOrigin: placement === 'bottom' ? 'center top' : 'center bottom',
-            }}
+
+        <FormControl component="fieldset" sx={{ mt: 0.5 }}>
```

Comment:
I believe we should also disable the radio buttons since the goal is to disable the interaction with the button.
```suggestion
        <FormControl component="fieldset" sx={{ mt: 0.5 }} disabled={disabled || loading}>
```